src/app/services/education/education.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import {Observable} from "rxjs";
+import {User} from "../../security/user";
+import {environment} from "../../../environments/environment";
+import {HttpClient} from "@angular/common/http";
+import * as educationActions from "../../store/actions/education.actions"
+import {Education} from "../../models/education";
+import {EducationPagination} from "../../models/education-pagination";
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EducationService {
+  api = environment.API_URL;
+  BASE_URL = this.api + '/educations';
+
+  constructor(private httpClient: HttpClient) { }
+
+  getEducationsByCandidateId(candidateId: string | null, page: number): Observable<EducationPagination> {
+    const url = `${this.BASE_URL}`;
+    console.log(url + '/candidateId=' + candidateId
+      + '/page=' + page);
+    return this.httpClient.get<EducationPagination>(url + '/candidateId=' + candidateId
```

Comment:
For string concatenation I would recommend using template literals. It makes your code more readable.
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals